packages/ui5-proxy-middleware/src/ui5/middleware.ts

```diff
@@ -21,7 +21,7 @@ module.exports = async ({ options }: MiddlewareParameters<Ui5MiddlewareConfig>):
     const router = express.Router();
     const config = options.configuration;
     const ui5Version = await resolveUI5Version(config.version, logger);
-    const secure = !!config.secure;
+    const secure = config.secure !== undefined ? !!config.secure : true;
```

Comment:
If it is null as opposed to undefined then it will not get a default value, is that what you want to do?

Reply:
It can be only `null` if the user decides to set it to `null`. So yes, it is intentional.

Reply:
🆗